net.shadps4.shadPS4.yaml

```diff
@@ -88,3 +91,10 @@ modules:
           url: https://api.github.com/repos/shadps4-emu/shadps4/releases/latest
           tag-query: .tag_name
           version-query: .tag_name | sub("^v."; "")
+      - type: script
+        dest-filename: net.shadps4.shadPS4.sh
+        commands:
+          - for i in {0..9}; do test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i; done
+          - shadps4
```

Comment:
Does this handle args from user? For example, when user want to run a game directly from the CLI.